ggml/cmake/ggml-config.cmake.in

```diff
@@ -129,10 +129,22 @@ if(NOT TARGET ggml::ggml)
         string(REPLACE "-" "_" _ggml_backend_pfx "${_ggml_backend}")
         string(TOUPPER "${_ggml_backend_pfx}" _ggml_backend_pfx)
 
-        find_library(${_ggml_backend_pfx}_LIBRARY ${_ggml_backend}
-            REQUIRED
-            HINTS ${GGML_LIB_DIR}
-            NO_CMAKE_FIND_ROOT_PATH)
+        if (GGML_BACKEND_DL)
+            # On systems with dynamically loaded backends, it's OK if not all are installed
+            find_library(${_ggml_backend_pfx}_LIBRARY ${_ggml_backend}
+                HINTS ${GGML_BACKEND_DIR} ${GGML_LIB_DIR}
+                NO_CMAKE_FIND_ROOT_PATH)
+
+            if (NOT ${_ggml_backend_pfx}_LIBRARY)
+                message(DEBUG "Built backend ${${_ggml_backend_pfx}_LIBRARY} not found")
+                continue()
+            endif()
+        else()
```

Comment:
I think this function could be skipped entirely when `GGML_BACKEND_DL` is enabled, right? Since it is not necessary to link to the libraries, everything is handled at run time.